app.js

```diff
@@ -79,6 +79,7 @@ $(document).ready(function() {
 
         });
 
+    // might be more readable to have this code run right after these 3 ajax objects are defined
     $.ajax(weatherAjax);
     $.ajax(quotesAjax);
     $.ajax(newsAjax);
```

Comment:
You also don't need to save it as a separate object, because the objects are so small. It makes code unnecessarily verbose. 

i.e. you can do the following for the weather AJAX call:
```
$.ajax({
  method: 'GET',
  url: 'http://api.wunderground.com/api/562b8535169e745a/conditions/q/CA/San_Francisco.json',
  success: wthrSuccess	
});
```

Reply:
^ I would also rename `wthrSuccess` to `weatherSuccess`. That way you don't need to define 3 different variables (`weatherUrl`, `weatherAjax`, `weatherSuccess`) to make this one call.